src/support.ts

```diff
@@ -26,9 +26,13 @@ export interface Support {
  */
 export function getSupport(feature: string, browser = currentBrowser): Support {
   const support: Support = { level: "unknown", needPrefix: false, notes: [] };
-  const stats = db.data[feature].stats[browser.id];
-  if (!stats) { return support; }
-
+  var feature = db.data[feature];
+  var browserId; 
+  if (feature && feature.stats) {
+      browserId = feature.stats[browser.id];
+  } else {
+      return support;
+  }
```

Comment:
The introduced `browserId` here is not used any further.
I think you meant sth like this:

```js
if (!db.data[feature] || !db.data[feature].stats[browser.id]) {
  return suppport;
}

const stats = db.data[feature].stats[browser.id];
```

Reply:
yes, it looks better. But in this case i will get an error (after just removing this line https://github.com/cssinjs/css-vendor/blob/master/test/fixtures/property-prefix.js#L68): `Uncaught ReferenceError: browserId is not defined`. Really, idk why.

Reply:
There is not `browserId` in the original code, did you forget a line in your PR somewhere?

Reply:
Yes, I changed it to make variable name more understandable

Reply:
Why do you want to remove https://github.com/cssinjs/css-vendor/blob/master/test/fixtures/property-prefix.js#L68? It's meant to be there, because caniuse-db didnt merged that into `data.json` and it still didn't. We could though add support specifically for those into this project until `caniuse-db` merges it (which they didn't do since a whole year). Then you can remove that line. Otherwise you'll construct a test case, whose data is not available, that can't work.